File: Programming Basics/JavaScript/Conditional Statements/Exercises/shopping.js
function solve(input) {
    const budget = Number(input[0]);
    const gpuCount = Number(input[1]);
    const cpuCount = Number(input[2]);
    const ramCount = Number(input[3]);

    const gpuCosts = gpuCount * 250;

    const cpuPrice = 0.35 * gpuCosts;
    const cpuCosts = cpuCount * cpuPrice;

    const ramPrice = 0.1 * gpuCosts;
    const ramCosts = ramCount * ramPrice;

    let totalCosts = gpuCosts + cpuCosts + ramCosts;
    if (gpuCount > cpuCount) totalCosts *= 0.85; // Apply 15% discount

    if (budget >= totalCosts) console.log(`You have ${(budget - totalCosts).toFixed(2)} leva left!`);
    else console.log(`Not enough money! You need ${(totalCosts - budget).toFixed(2)} leva more!`);
}

File: Programming Basics/C++/Conditional Statements/Exercises/Shopping/Shopping.cpp
#include <iostream>
using namespace std;

int main()
{
    double budget;
    int gpuCount, cpuCount, ramCount;
    cin >> budget >> gpuCount >> cpuCount >> ramCount;

    double gpuCosts = gpuCount * 250;

    double cpuPrice = 0.35 * gpuCosts;
    double cpuCosts = cpuCount * cpuPrice;

    double ramPrice = 0.1 * gpuCosts;
    double ramCosts = ramCount * ramPrice;

    double totalCosts = gpuCosts + cpuCosts + ramCosts;
    if (gpuCount > cpuCount)
        totalCosts *= 0.85; // Apply 15% discount

    cout.setf(ios::fixed);
    cout.precision(2);

    if (budget >= totalCosts) cout << "You have " << budget - totalCosts << " leva left!" << endl;
    else cout << "Not enough money! You need " << totalCosts - budget << " leva more!" << endl;
}
